Documentation.md

```diff
@@ -0,0 +1,207 @@
+Welcome! You can find here documentation for /r/oneplus Discord server bot.
+
+Its prefix is `;` but commands can also be triggered by pinging bot.\
+Note that commands only works in guilds.
+
+# Features:
+
+## Owner
+
+### Emergency bot shutdown
+
+`;x`\
+Terminates bot process.
+
+### SQL commands
+
+`;sql [insert SQL command]`\
+Allow owner to edit content of SQL databases directly via a Discord message.
+
+### Update
+
```

Comment:
add command
add description about updating the roles and the channels currently present on the server (thats what the command does)